packages/design/src/Spacing.mdx

```diff
@@ -6,7 +6,89 @@ route: /space
 
 # Spacing
 
-Spacing for the Jobber Atlantis Design System
+Our spacing system exists to support the foundational visual design principle of
+[proximity-based grouping:](https://www.nngroup.com/articles/gestalt-proximity/)
+
+> Using varying amounts of whitespace to either unite or separate elements is
+> key to communicating meaningful groupings.
+
+In the shortest, simplest terms: the closer two elements are related, the
+tighter the spacing between them should be.
+
+With in mind, let’s start small and work our way up.
+
+## Design & usage guidelines
+
+### Miniscule and Smallest
+
+At their respective sizes (`1px` and `2px` equivalent), these values should
+mostly be used to provide optical adjustments where an element is visually
+off-centered.
+
+### Smaller
+
+The `smaller` spacing value can be used between contents of a single component.
+For example, see the spacing between an Icon and the label inside of a
+[Button.](/components/button#icons)
+
+![illustration of a smaller space between icon and button label](https://user-images.githubusercontent.com/39704901/143141489-679fa61b-5497-4e81-bfe0-b5f463673e4b.png)
+
+### Small
+
+Sibling items within a container can be separated by the `small` value to
+reflect their relation to each other. An example would be the spacing between
+[Chips in a selection group.](/components/chips#single-select)
+
+![illustration of a small space between Chip components](https://user-images.githubusercontent.com/39704901/143141613-193ceeb2-b393-44ab-a5e8-21d2f88c67c4.png)
+
+### Base
+
+Use `base` as the default spacing in larger content containers with multiple
+children of their own, such as [Card.](/components/card) This should also be
+used as the default starting point for spacing between [Form](/components/form)
+elements.
+
+![illustration of spacing between form elements inside of a card](https://user-images.githubusercontent.com/39704901/143141662-0f20eec4-8b27-44fe-96f5-1bf1c4d567aa.png)
+
+### Large
```

Comment:
@chriscarline I _think_ this is more specific but still generalizable? Lemme know what you think.

Reply:
Preview: https://deploy-preview-779--jobber-atlantis.netlify.app/space#large

Reply:
@chris-at-jobber makes sense to me! Just need to drop those two its'

Reply:
✅